sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java

```diff
@@ -1548,14 +1548,27 @@ public Node visitWhenClause(SqlBaseParser.WhenClauseContext context) {
 
     @Override
     public Node visitFunctionCall(SqlBaseParser.FunctionCallContext context) {
+        var window = visitIfPresent(context.over(), Window.class);
+        var filter = visitIfPresent(context.filter(), Expression.class);
+        if (filter.isPresent() && window.isPresent()) {
```

Comment:
or better to do it later?

Reply:
Yes, see my comment. Didn't see your comment in the first place ;)

Reply:
makes sense! removing